spec/Section 6 -- Execution.md

```diff
@@ -143,6 +147,88 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
 
+### Subscription
+
+Unlike queries and mutations, subscriptions have a lifetime that consists of three
+phases. Between the the subscribe and unsubscribe phases, the subscription is
+considered to be in the "active" state.
+
+#### Subscribe
+
+The result of executing a subscription operation is a subscription object with
+the following capabilities:
+
+  * **Must** support observation of the associated publish stream (for example,
+  via iteration, callbacks, or reactive semantics).
+  * **Must** support cancellation of the subscription, see {Unsubscribe}.
+  * **Must** support a way to identify the subscriber/subscription pair (for
+    example, a GUID/callback table or closure over the callback).
+  * **May** include an initial response associated with executing the selection
+  set defined on the subscription operation.
+
+Subscribe(schema, subscription, operationName, variableValues, initialValue):
+
+  * Let {subscriptionType} be the root Subscription type in {schema}.
+  * Assert: {subscriptionType} is an Object type.
+  * Let {selectionSet} be the top level Selection Set in {subscription}.
+  * Let {rootField} be the first top level field in {selectionSet}.
+  * Let {eventStream} be the result of running {MapSubscriptionEvents(rootField, variableValues)}.
+  * Optionally: let {data} be the result of running {ExecuteRequest(schema, document, operationName, variableValues, initialValue)}, and return {data} on the subscription object.
```

Comment:
@robzhu and I talked about this offline, but I just wanted to point out that this initialValue addition improves but doesn't guarantee a solution to race conditions. Events can still potentially be repeated or dropped between the generation of the initial value payload and subsequent payloads, depending on the system design and the latency of events flowing through it.

I think the kicker here is that most examples of wanting initial data before subscription payloads desire a *different* query shape - in the example above a list of comments vs an individual comment. The spec as written wouldn't allow for this distinction, so I actually think your example is more powerful while requiring less from the spec.